packages/cli/src/networks/index.ts

```diff
@@ -77,6 +78,64 @@ export async function fetchBootnodes(network: NetworkName): Promise<string[]> {
   return enrs;
 }
 
+/**
+ * Reads and parses a list of bootnodes for a network from a file.
+ */
+export function readBootnodes(bootnodesFilePath: string): string[] {
+  const bootnodesFile = fs.readFileSync(bootnodesFilePath, "utf8");
+
+  const bootnodes = parseBootnodesFile(bootnodesFile);
+
+  if (bootnodes.length === 0) {
+    throw new Error(`No bootnodes found on file ${bootnodesFilePath}`);
+  }
```

Comment:
It looks like there's also a check for 0 bootnodes in `enrsToNetworkConfig`

Reply:
Yup, it's not required anymore with the previous changes. Removed in https://github.com/ChainSafe/lodestar/pull/3562/commits/f20a640b0cf7189909d96feff8bab456856fa8d5